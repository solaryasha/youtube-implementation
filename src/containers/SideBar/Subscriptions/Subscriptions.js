import React, { Fragment } from 'react';
import SideBarHeader from '../SideBarHeader/SideBarHeader';
import Subscription from './Subscription/Subscription';
import { Divider } from 'semantic-ui-react';

const Subscriptions = (props) => {
    return (
        <Fragment>
            <SideBarHeader title='subscriptions' />
            <Subscription label='MusicChannel' broadcasting />
            <Subscription label='Coursea' newVideoAmount="10" />
            <Subscription label='TEDx Talks' newVideoAmount="23" />
            <Subscription label='Stanford iOS' newVideoAmount="4" />
            <Subscription label='Udacity' newVideoAmount="114" />
            <Divider />
        </Fragment>
    )
}

export default Subscriptions;