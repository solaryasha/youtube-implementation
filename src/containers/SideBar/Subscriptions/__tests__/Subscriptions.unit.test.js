import React from 'react';
import {shallow} from 'enzyme';
import Subscriptions from '../Subscriptions';

test('subscriptions rendering', () => {
    const wrapper = shallow(
        <Subscriptions />
    );
    expect(wrapper).toMatchSnapshot();
})