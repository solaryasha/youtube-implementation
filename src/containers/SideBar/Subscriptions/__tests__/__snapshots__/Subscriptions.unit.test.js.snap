// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`subscriptions rendering 1`] = `
<Fragment>
  <SideBarHeader
    title="subscriptions"
  />
  <Subscription
    broadcasting={true}
    label="MusicChannel"
  />
  <Subscription
    label="Coursea"
    newVideoAmount="10"
  />
  <Subscription
    label="TEDx Talks"
    newVideoAmount="23"
  />
  <Subscription
    label="Stanford iOS"
    newVideoAmount="4"
  />
  <Subscription
    label="Udacity"
    newVideoAmount="114"
  />
  <Divider />
</Fragment>
`;
